spring-restdocs-core/src/main/java/org/springframework/restdocs/cli/CurlRequestSnippet.java

```diff
@@ -45,6 +44,8 @@
  */
 public class CurlRequestSnippet extends TemplatedSnippet {
 
+	private CommandFormatter commandFormatter = ConcatenatingCommandFormatter.newlineSeparator();
+
 	/**
 	 * Creates a new {@code CurlRequestSnippet} with no additional attributes.
 	 */
```

Comment:
Just a quick question about this one: deprecating this constructor would mean that I would have to touch every test in `CurlRequestSnippetTests` to switch to a different constructor. Is that ok for you?